src/components/css/addBooks.css

```diff
@@ -0,0 +1,67 @@
+.FormContianer-all {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.FormContianer {
+  border-top: 1px solid white;
+  display: flex;
+  flex-direction: column;
+  height: 10.625rem;
+  margin: 1.938rem 0.063rem 2.5rem 0;
+  border-radius: 4px;
+}
+
+.title {
+  display: flex;
+  text-align: left;
+  color: #666;
+  font-weight: bold;
+}
+
+.form {
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+}
+
+.input-container {
+  display: flex;
+}
+
+.input-container:nth-child(1) {
+  flex: 3;
+}
+
+.input-container:nth-child(2) {
+  flex: 2;
+}
+
+.input-container:nth-child(3) {
+  flex: 1;
+}
+
+.input-container input {
+  width: 100%;
+  height: 2.813rem;
+  background-color: white;
+  border: 1px solid lightgray;
+  padding-left: 10px;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.15px;
+  color: #c4c4c4;
+}
+
+.input-container button {
+  background-color: #0290ff;
+  border: none;
+  width: 100%;
+  color: #fff;
+  font-size: 20px;
+  border-radius: 3px;
+}
```

Comment:
- [ ] Good job for styling the button 🥇 However, as mentioned [here](https://github.com/AliShahSafdari/bookstore/pull/5#discussion_r1231113342), please make sure to style the button as it is given in the design guidelines ✨✌ Just a little bit of work is needed. See the images below for clarity 💡
- Please note that the `font-family` used for button is `RobotoSlab`. Also, the text is in `uppercase` 🤓⛏ You can checkout the other properties from Zeplin design, Kindly fix this

Your design | Given design 
---| ---|
![image](https://github.com/AliShahSafdari/bookstore/assets/106119105/7d00addf-da56-43a0-b9cb-e200fb8604d2) | ![image](https://github.com/AliShahSafdari/bookstore/assets/106119105/b6662843-30bb-43f6-bee5-761dc1a3ad36)